packages/roosterjs-editor-types/lib/interface/FormatState.ts

```diff
@@ -84,6 +85,11 @@ export interface ElementBasedFormatState {
      */
     canAddImageAltText?: boolean;
 
+    /**
+     * Format of image, if there is table at cursor position
+     */
+    imageFormat?: ImageFormat;

```

Comment:
For other format sub types, they are all supported by both Content model and original API. But this one is only supported by Content Model. In that case, I think we can have a sub type of FormatState in Content Model package to hold image format. So when people use original API, they will only see original formats.

What do you think?